selfdrive/updated.py

```diff
@@ -112,10 +113,10 @@ def set_params(new_version: bool, failed_count: int, exception: Optional[str]) -
 
   if new_version:
     try:
-      with open(os.path.join(FINALIZED, "RELEASES.md"), "rb") as f:
+      with open(os.path.join(FINALIZED, "RELEASES.md"), "r") as f:
         r = f.read()
-      r = r[:r.find(b'\n\n')]  # Slice latest release notes
-      params.put("ReleaseNotes", r + b"\n")
+      r = r[:r.find('\n\n')]  # Slice latest release notes
+      params.put("ReleaseNotes", markdown.markdown(r, tab_length=2))
```

Comment:
Can you add another layer of exception handling here, and put in the raw release notes in case `markdown.markdown()` throws an exception?